test/fibonacci.js

```diff
@@ -0,0 +1,16 @@
+import { expect } from 'chai'
+import fibonacci from '../src/fibonacci'
+
+describe.only('fibonacci()', function(){
+
+  it('should be a function', function(){
+    expect(fibonacci).to.be.a('function')
+  })
+
+  it('should return an array of length 50, whos 9th value is 34', function(){
+    const fib = fibonacci(50)
+    expect(fib).to.be.an('array')
+    expect(fib.length).to.equal(50)
+    expect(fib[9]).to.equal(34)
```

Comment:
I would assert at least the first 10 or even 20 places of the fibonacci sequence.